apps/backend/src/libs/modules/controller/libs/types/api-handler-options.type.ts

```diff
@@ -6,6 +8,7 @@ type APIHandlerOptions<
 	T extends DefaultApiHandlerOptions = DefaultApiHandlerOptions,
 > = {
 	body: T["body"];
+	originalRequest?: FastifyRequest | undefined;
```

Comment:
After we turn off a specificed option in tsconfig, please, remove undefined from here as it is redundant